src/_scss/pages/keyword/header/header.scss

```diff
@@ -1,39 +1,46 @@
-@import "layouts/tabbedSearch/header/header";
+@import "layouts/default/stickyHeader/header";
 
-.search-header-container {
-    .search-header {
-        .search-title {
+.sticky-header__container {
+    .sticky-header {
```

Comment:
The block should not be defined inside one of its member element.
Additionally, it seems like you are making a variant of the `sticky-header` block here for the keyword header bar. Since the `<StickyHeader />` component utilizes children to wrap around the header contents, maybe a better approach would be to make the keyword header bar's contents its own module, that handles the inner layout and formatting independently of the parent `sticky-header` block.